lib/src/widgets/chat/ds_select_menu.widget.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:convert';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSSelectMenu extends StatelessWidget {
+  final DSAlign align;
+  final Map<String, dynamic> content;
+  final Function? onSelected;
+
+  const DSSelectMenu({
+    Key? key,
+    required this.align,
+    required this.content,
+    this.onSelected,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 18.0),
+      child: Column(
+        children: _buildSelectMenu(),
+      ),
+    );
+  }
+
+  List<Widget> _buildSelectMenu() {
+    final List<Widget> children = [];
+
+    int count = 0;
+
+    content['options'].forEach(
+      (dynamic option) {
```

Comment:
**OBS: Same comment in the Quick Reply file.**

If makes sense, this ```option``` variable could have a model class instead of using a ```dynamic``` type.

Reply:
done